docs/advanced.md

```diff
@@ -1018,7 +1018,7 @@ UI.error "Something unexpected happened in my_new_action. Attempting to continue
 |verbose|Displays verbose output in the default output color|
 |header|Displays a message in a box for emphasis|
 
-Methods ending in and exclamation point (`!`) terminate execution of the current
+Methods ending in an exclamation point (`!`) terminate execution of the current
```

Comment:
Shouldn't this be `exclamation mark`?

Reply:
I think "exclamation point" and "exclamation mark" are synonymous. This is more natural to me, but I think either one is acceptable.

Reply:
I'm not a native english speaker and I only knew about "exclamation mark", so then disregard this review.